tests/containers/podman_firewall.pm

```diff
@@ -32,24 +33,24 @@ sub run {
     }
 
     # cni-podman0 interface is created when running the first container
-    assert_script_run "podman pull " . registry_url('alpine');
-    assert_script_run "podman run --rm " . registry_url('alpine');
+    assert_script_run "$podman pull " . registry_url('alpine');
```

Comment:
Does this work? This is now an object which doesn't return any string `podman` or anything like this. The reason to have this object is to :
1) install & config registries
2) optionally use the subroutines defined in the Class

So you can do things like `$podman->methodXYZ()` but you can't use it as replacement for `podman` as a string.

Either you keep `podman pull` or you do `$podman->_engine_assert_script_run('pull XYZ')`.

Reply:
This works and this is valid. I will not use those `_engine` subroutines as they are slowly getting deprecated.

Reply:
ok, but using the podman object like this looks strange... since this module is called `podman_something`, you can just use `podman` as it was before.

Reply:
OK. I'll just pet it back :smile:.

But when you have `$engine` you don't have to do `$engine->engine`.